lib/Frontend/nvqpp/ConvertExpr.cpp

```diff
@@ -1083,6 +1083,33 @@ bool QuakeBridgeVisitor::VisitCallExpr(clang::CallExpr *x) {
                   builder, ext->getLoc(), 0,
                   ext.getResult().getType().getIntOrFloatBitWidth())));
         }
+    if (funcName.equals("front"))
+      if (auto memberCall = dyn_cast<clang::CXXMemberCallExpr>(x))
+        if (memberCall->getImplicitObjectArgument()) {
+          [[maybe_unused]] auto calleeTy = popType();
+          assert(isa<FunctionType>(calleeTy));
+          auto zeroIndex = getConstantInt(builder, loc, 0, 64);
+          auto eleTy = cast<cc::StdvecType>(svec.getType()).getElementType();
+          auto elePtrTy = cc::PointerType::get(eleTy);
+          auto vecPtr = builder.create<cc::StdvecDataOp>(loc, elePtrTy, svec);
+          auto eleAddr = builder.create<cc::ComputePtrOp>(
+              loc, elePtrTy, vecPtr, ValueRange{zeroIndex});
+          return pushValue(builder.create<cc::LoadOp>(loc, eleAddr));
```

Comment:
```suggestion
          auto eleTy = cast<cc::StdvecType>(svec.getType()).getElementType();
          auto elePtrTy = cc::PointerType::get(eleTy);
          return pushValue(builder.create<cc::StdvecDataOp>(loc, elePtrTy, svec));
```
front can be simplified since adding an offset of 0 is unnecessary.